vk_requests/session.py

```diff
@@ -170,6 +175,78 @@ def do_oauth2_authorization(self, session):
         logger.error('Permissions obtained')
         raise VkAuthError(error_message)
 
+    def do_direct_authorization(self, session):
+        """ Direct Authorization, more info: https://vk.com/dev/auth_direct """
+        logger.info('Doing direct authorization, app_id=%s', self.app_id)
```

Comment:
Also "Direct authorization" sounds a bit too general and may be confusing (at least to me). 
Let's mention "Client Credentials Flow" in the message as well

Reply:
`Client Credentials Flow` was used to access secure methods and was deprecated. `Direct Authorization` is completely different.

Reply:
Oh, alright. Then we can keep it as it is. Link to the documentation in the method description is helpful enough